.gitignore

```diff
@@ -8,3 +8,4 @@ _site/
 /vendor
 /example/*/build
 /.venv
+/node_modules
```

Comment:
Why was this suddenly needed?

Reply:
ah yeah, i had added some mermaid node stuff for diagramming but pulled it out :facepalm: will revert! tyty